### DISCUSSION TITLE
Welcome to Control Surface Discussions!

### DISCUSSION THREAD

**Discussion by tttapa** [tttapa/Control-Surface#354] [General]
## 👋 Welcome!
  Control Surface uses Discussions as a place to connect with other members of the community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded.

### Guidelines

When possible, **always post the full code** you're having trouble with. Prefer a short, [minimal reproducible example](https://stackoverflow.com/help/minimal-reproducible-example), don't post hundreds of lines of code.

If you get any compilation errors or warnings, post them as well (don't just describe them in your own words, copy and paste the exact error messages).

Finally, include the name of the Arduino board you're using, the version of the Arduino Core you have installed for that board (see Tools > Board > Boards Manager in the IDE, search for your board and report the version) and the version of Control Surface or any other libraries you may be using.

When posting code or error messages, please use three backticks with a language identifier:

~~~
```c++
// Your code here
```
~~~

If you're not familiar with GitHub's MarkDown syntax for formatting messages, you can read more here: https://guides.github.com/features/mastering-markdown

**Comment by officialpalash** [comment]
How to press shift key and other note buttons
Like dj controllers shift key
How to code shift key

**Reply by officialpalash** [reply]
Please reply

**Reply by tttapa** [reply]
See https://tttapa.github.io/Control-Surface-doc/Doxygen/da/dc1/FAQ.html#faq-change-address-runtime
